deepchem/models/torch_models/tests/test_hnn.py

```diff
@@ -0,0 +1,25 @@
+import torch
+import pytest
+from deepchem.models.torch_models import HNN
+
+
+
+@pytest.fixture
```

Comment:
I checked the CI, and it seems this is causing some linting issues (yapf and flake8). Can you please fix and check linting tests run correctly?